src/App.jsx

```diff
@@ -1,44 +1,20 @@
 import React from 'react';
 import 'bulma/css/bulma.css';
 
+import { Article } from './components/Article';
```

Comment:
remove unnecessary line
```suggestion
import { Article } from './components/Article';
```